eu.ad5001.LogarithmPlotter.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "eu.ad5001.LogarithmPlotter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
```

Comment:
```suggestion
    "runtime-version": "5.15-21.08",
```

Reply:
There seems to be an issue with building PySide2 under the KDE SDK & runtime v5.15-21.08 where CLang isn't being properly detected anymore. I'll revert back to v5.15 in the meantime. 
```
CMake Error at sources/shiboken2/data/shiboken_helpers.cmake:146 (message):
  Unable to detect CLANG location by checking LLVM_INSTALL_DIR,
  CLANG_INSTALL_DIR or running llvm-config.
Call Stack (most recent call first):
  sources/shiboken2/CMakeLists.txt:39 (setup_clang)
```